control.tex

```diff
@@ -1,38 +1,33 @@
 \chapter{Encoder Control} \label{encoderControl}
 
-This chapter describes the fields required to control the Trace Encoder.  
+The fields required to control a Trace Encoder are defined in the 
+\href{https://github.com/riscv-non-isa/tg-nexus-trace/blob/master/docs/RISC-V-Trace-Control-Interface.adoc}{RISC-V Trace Control Interface Specification},
+which is intended to apply to any and all RISC-V trace encoders, regardless of encoding protocol.

```

Comment:
Is this too broad in claiming applicability to any and all encoders for all times? 
"which applies to RISC-V trace encoder that uses the E-trace specified encoding protocol"?

Reply:
No - the whole point about the common control specification is that it is encoding protocol agnostic.  Currently it applies to E-Trace and N-Trace, and the intent is that it will apply for any other future encoders if there are any.